pages/apim/3.x/installation-guide/configuration/gateway/opentracing-in-docker.adoc

```diff
@@ -1,92 +1,26 @@
-= OpenTracing In Docker
+= Enable OpenTracing
```

Comment:
This page should only be about Jaeger configuration and not docker.
all this content should be moved to the "How to / Tuto" page